content/en/keps/_index.md

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
keps should not be a top level item. I think it'd make more sense under resources.

Reply:
I will work on it.